menu/additional_apps.sh

```diff
@@ -36,8 +36,12 @@ case "$choice" in
         clear
         print_text_in_color "$ICyan" "Downloading the Documentserver script..."
         run_script MENU documentserver
+   *"Bitwarden "*)
+        clear
+        print_text_in_color "$ICyan" "Downloading the Bitwarden script..."
+        run_script MENU bitwarden_menu
     ;;&
-    *"Fail2ban"*)
+    *"Fail2ban "*)
```

Comment:
needs fixing

Reply:
This was actually by design. because if you don't have the space in there, if you choose Fail2ban-Statuscheck, it will run also the Fail2Ban script...

Reply:
Aah, ok, please revert.

Reply:
okay, is reverted